packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -233,7 +233,7 @@ export const TooltipPlugin: React.FC<TooltipPluginProps> = ({
       tooltip = <SeriesTable series={series} timestamp={xVal} />;
     }
   } else {
-    tooltip = renderTooltip(otherProps.data, focusedSeriesIdx, focusedPointIdx);
+    tooltip = renderTooltip(otherProps.data, focusedSeriesIdx, plotInstance.current?.cursor.idx ?? focusedPointIdx);
```

Comment:
this looks unrelated? (and breaks State timeline tooltip)

http://localhost:3000/d/mIJjFy8Gz/timeline-modes?orgId=1